test/Pass/syn-inst-sadd-sat.ll

```diff
@@ -0,0 +1,19 @@
+; REQUIRES: solver
```

Comment:
you should try to add diversity in test cases
if the souper test used one way to detect overflow, use a different one here
if the souper test is 8 bits, make this one 16 or 32
etc.
you're trying to dig up bugs here, doing different things in different ways is part of the goal